### DISCUSSION TITLE
Would it be possible to use compile-time reflection?

### DISCUSSION THREAD

**Discussion by sschuberth** [LemonAppDev/konsist#425] [Ask a Question]
### Wanted end result.

Replace kotlin-reflect with a faster alternative.

### What you tried so far.

I'm considering https://github.com/JetBrains-Research/reflekt.

**Comment by igorwojda** [comment]
Thank you for letting me know about `reflect`. It looks like `"org.jetbrains.kotlin:kotlin-reflect` was a leftover and it was removed https://github.com/LemonAppDev/konsist/pull/426.
